kubernetes/client/ws_client.py

```diff
@@ -12,9 +12,11 @@
 
 from .rest import ApiException
 
+import select
 import certifi
+import time
 import collections
-import websocket
+from websocket import WebSocket, ABNF, enableTrace
```

Comment:
we have pep8 and sort_import running automatically and checking these. We cannot organize them the way we want unless we want to remove those checks.